sys/hacl/c/config/vale-aes.h

```diff
@@ -5,4 +5,5 @@
 
 #include "config.h"
 
-#include "Vale.h"
+#include "EverCrypt_AutoConfig2.h"
```

Comment:
How much work would it be to sidestep evercrypt for AES GCM?
That's the only one that doesn't have a HACL API.

Reply:
I hate using Evercrypt as much as you, let's keep it on TODO list and get back to it when we have time for.

Reply:
#66